src/routes/imgurRoutes.js

```diff
@@ -0,0 +1,73 @@
+const userRoutes = require('./userRoutes');
+const express = require('express');
+const fileUploadMiddleware = require('express-fileupload');
+const { ImgurClient } = require('imgur');
+const User = require('../models/User');
+
+const router = express.Router();
+const imgurClient = new ImgurClient({
+  clientId: process.env.IMGUR_CLIENT_ID,
+  clientSecret: process.env.IMGUR_CLIENT_SECRET,
+  refreshToken: process.env.IMGUR_REFRESH_TOKEN,
+});
+
+userRoutes.post(
```

Comment:
If we're defining this on the `userRoutes` then let's put this file in the `userRoutes` file instead of spreading across two files?

Reply:
Since you've defined this route on `userRoutes`, that means that Express only knows to look for `/:userId/uploadProfile` when it's after `api/v1/users`. 

I think that's totally fine, but in that case, this doesn't need to be here in a separate file, we should instead be defining this alongside the rest of the `api/v1/users` routes in the `userRoutes.js` file

![](https://github.com/Code-the-Dream-School/dd-prac-team4-back/assets/9094098/cb22e619-858b-41e8-a632-0920eeee10f9)

Reply:
imgur routes moved to UserRoutes, commit 0cb57b38cc269ee57c85dcee133abe92f8bfcf10